includes/Stories.php

```diff
@@ -0,0 +1,180 @@
+<?php
```

Comment:
Why does this live in this namespace. How and where will be used? Should this in it's own namespace?

Also the naming of this class, implements it generates an array of story objects using a story model.

Reply:
Would the naming be better as `Story_Query`